src/detections/filters/accordion-details/class/index.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { Grid } from '@material-ui/core'
+
+import { FilterWrapper } from 'detections/filters/accordion-details/filter-wrapper'
+import { RemoveButton } from 'detections/filters/accordion-details/remove-button'
+import { Input } from 'detections/filters/accordion-details/class/input'
+import { FilterTitle } from 'detections/filters/accordion-details/filter-title'
+
+export const Class = index => <FilterWrapper>
+  <FilterTitle title={'Class'} />
+  <Input />
+  <Grid container item xs={1} sm={2} md={2} lg={2} xl={2} justify={'flex-end'} alignContent={'center'}>
```

Comment:
Grids cannot be both an item and a container at the same time. Let's split this into separate grids.

The same applies for the others as well.

Reply:
done